src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -709,6 +708,104 @@ transactionFormatResultImpl(Transaction::pointer tpTrans, unsigned apiVersion)
 
 //------------------------------------------------------------------------------
 
+XRPAmount
+getBaseFee(Application const& app, Config const& config, Json::Value tx)
+{
+    if (!tx.isMember(jss::Fee))
+    {
+        tx[jss::Fee] = "0";
+    }
+    if (!tx.isMember(jss::Sequence))
+    {
+        tx[jss::Sequence] = "0";
+    }
+    if (!tx.isMember(jss::SigningPubKey))
+    {
+        tx[jss::SigningPubKey] = "";
+    }
+    if (!tx.isMember(jss::TxnSignature))
+    {
+        tx[jss::TxnSignature] = "";
+    }
+    if (tx.isMember(jss::Signers))
+    {
+        if (!tx[jss::Signers].isArray())
+            return config.FEES.reference_fee;
+        // check multisigned signers
+        for (auto& signer : tx[jss::Signers])
+        {
+            if (!signer.isMember(jss::Signer) ||
+                !signer[jss::Signer].isObject())
+                return config.FEES.reference_fee;
+            if (!signer[jss::Signer].isMember(sfTxnSignature.jsonName))
+            {
+                // autofill TxnSignature
+                signer[jss::Signer][sfTxnSignature.jsonName] = "";
```

Comment:
Signer.TxnSignature should be set to "", but it is not actually set.

Checked by inserting the following code.

```cpp
std::cout << "Check Signers: " << tx << "\n";
```

Reply:
There is a test case for this: https://github.com/XRPLF/rippled/pull/5069/files#diff-a426c2d08805c8473aa7e6c95fc43c262a28309ceda68664d5b7276e557ffe8eR637-R644
It would error in the transaction template-checking if it failed here. Could you give me the tx JSON you're feeding it?

Reply:
This should not pass the part I am pointing out because in simualte RPC the Signer.TxnSignature is covered by the following part.

https://github.com/XRPLF/rippled/blob/c94c1374c45ce1fccec79aadff167057cb2f3ed3/src/xrpld/rpc/handlers/Simulate.cpp#L60-L83

Reply:
Can you provide an example input that results in this issue?

Reply:
Although not a simple reproduction code, I found this issue in the following test with code modified to use getBaseFee for Fee RPC.

https://github.com/tequdev/rippled/blob/db7648040b40400ef758429bfa744f4b6ede9d97/src/test/app/BaseFee_test.cpp#L189-L190

https://github.com/tequdev/rippled/blob/db7648040b40400ef758429bfa744f4b6ede9d97/src/xrpld/rpc/handlers/Fee1.cpp#L93

Reply:
This breaks existing code?

Reply:
I investigated again and found that this was a problem with my code.
Sorry.